### PR TITLE
Build about us section

### PR DESCRIPTION
**Pull request by fahdrahali**
**Create the about us section in the file Html and make a style:**

- Use FlexBox
- Use semantic HTML
- Create a Form
- test the form, send data

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -102,6 +102,28 @@ <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
             </div>
     </section>
     </main>
+    <footer class="flex-footer"> 
+       <h3 class="title">I'm always interested in hearing about new projects, 
+           so if you'd like to chat please get in touch.</h3>
+        <form class="flex-form" action="https://formspree.io/f/mqknqpyg" method="post" >
+            <input type="text" name="fullname" placeholder="Full name" maxlength="30" required>
+            <input type="email" name="email" placeholder="Email address" required>
+            <textarea name="comment"
```
Comment (micheaol) [inline index.html:111]: Please ensure that message input has maxlength="500" attribute according to the requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @fahdrahali , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️

- _Check the comments under the review._
- The commit history shows that you are the only contributor to this project. Please is there any reason for this? Because this should pair-programming project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @fahdrahali, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] As required by the previous reviewer, please make sure there is a commit from your partner as this is a pair-programming exercise.

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! Kindly tag me using @AnselemOdims for a quicker response.

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @fahdrahali ** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 
- Descriptive README file ✔️ 
- Descriptvie PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**